packages/application-generic/src/usecases/select-integration/select-integration.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Test } from '@nestjs/testing';
+import { ChannelTypeEnum, EmailProviderIdEnum } from '@novu/shared';
+import { IntegrationEntity, IntegrationRepository } from '@novu/dal';
+import * as sinon from 'sinon';
+import { UserSession } from '@novu/testing';
+import { SelectIntegration } from './select-integration.usecase';
+import { SelectIntegrationCommand } from './select-integration.command';
+
+describe('select integration', function () {
```

Comment:
the tests are not working if I do run them manually with `node 'node_modules/jest/bin/jest.js' './src/usecases/select-integration/select-integration.spec.ts' -c './jest.config.js' -t 'select integration'`

Reply:
All the tests in the `application-generic` package are failing with the errors with the assertion methods
`Property 'toContain' does not exist on type 'Assertion'. Did you mean 'contain'?`

Reply:
yes, but let's make sure that this one works, so someone who will be fixing them will have less work to do